src/_services/comment.service.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+import { authHeader } from '../_helpers';
+
+export const commentService = {
+    add,
+    update,
+    delete: _delete,
+    getAll,
+};
+let url = process.env.VUE_APP_DJANGO_HOST;
+function getAll(food_id) {
+    return axios
+        .get(`${url}/comments/?food_id=${food_id}`)
```

Comment:
sao phải có `?` rồi gán params `food_id` e? `/comment/${food_id}` ko dc à?

Reply:
i dont know wwhat u do, but if u wwant to use req.query,  should remove / before ? and req.params as well